packages/newspringchurchapp/src/prayer/PrayerActionMenuCard/PrayerActionMenuCardConnected.js

```diff
@@ -3,13 +3,27 @@ import PropTypes from 'prop-types';
 import { Mutation } from 'react-apollo';
 
 import savePrayerRequest from '../data/mutations/savePrayerRequest';
+import savedPrayerList from '../data/queries/getSavedPrayers';
```

Comment:
```suggestion
import GET_SAVED_PRAYERS from '../data/queries/getSavedPrayers';
```

Reply:
Holding off on this until Vincent speaks his peace.